app/src/tests/mocks/mockResponses.ts

```diff
@@ -0,0 +1,30 @@
+export const processFilesMockResponseNonConflict = {
+  result: "file"
+};
+
+export const processFilesMockResponseConflict = {
+  specs: [
+    "file1",
+    "file2"
+  ],
+  driverFile: "file",
```

Comment:
I thought you guys decided to call this the "defaults" file

Reply:
Will sync with Rafi later today